### PR TITLE
UI: added new set of functionalities

### PR DESCRIPTION
**Pull request by aanm**
Signed-off-by: André Martins andre@cilium.io

### CODE REVIEW

File: daemon/server/handlers.go [hunk 1]
```diff
@@ -444,7 +444,30 @@ func (router *RouterBackend) policyAdd(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusCreated)
 }
 
-func (router *RouterBackend) policyDelete(w http.ResponseWriter, r *http.Request) {
+func (router *Router) policyAddForm(w http.ResponseWriter, r *http.Request) {
+	err := r.ParseMultipartForm(32 << 20)
```
Comment (tgraf) [inline daemon/server/handlers.go]: What does this magic value do? :8ball:

**Comment by tgraf** [comment]
W00t, we have a loading gif. Code LGTM aside from the minor build failure.

**Comment by tgraf** [comment]
Providing ACK given build is fixed and magic value is explained.

File: daemon/server/handlers.go (hunk 1 above)
Comment (aanm) [inline daemon/server/handlers.go]: It's the max memory for that go can use to parse the form. Fixed!

File: daemon/server/ui.go [hunk 2]
```diff
@@ -167,7 +36,15 @@ var (
 	}
 )
 
-func (router *RouterUI) createUIHTMLIndex(w http.ResponseWriter, r *http.Request) {
+func init() {
```
Comment (aanm) [inline daemon/server/ui.go:39]: @tgraf I had to add this `init()` so the tests wouldn't fail.

**Pull request merged**